src/odb/include/odb/dbStream.h

```diff
@@ -222,6 +226,21 @@ class dbOStream
     return *this;
   }
 
+  template <uint32_t I = 0, typename... Ts>
+  dbOStream& operator<<(const std::variant<Ts...>& v)
+  {
+    static_assert(I < kTemplateRecursionLimit);
```

Comment:
Let's add a message to the static assert. "OpenROAD disallows of std::variants larger than 16 elements. You should look into alternate solutions."

Also we probably want <=.